tests/unittests/CoreGraphics/CGPathTests.mm

```diff
@@ -871,7 +871,31 @@ static bool testSymmetricEquivalence(CGPathRef path1, CGPathRef path2) {
     CGPathRelease(containedRectangle);
 }
 
-DISABLED_TEST(CGPath, SubShapesEqualityTest) {
+TEST(CGPath, CGPathEqualsCopyTest) {
+    CGMutablePathRef thepath = CGPathCreateMutable();
+    CGFloat width = 500.0f;
+    CGFloat height = 500.0f;
+    CGFloat xstart = 50.0f;
+    CGFloat ystart = 50.0f;
+
+    CGPathMoveToPoint(thepath, NULL, xstart + .75 * width, ystart + .5 * height);
+    CGPathAddArc(thepath, NULL, xstart + .5 * width, ystart + .5 * height, .5 * height, 0, M_PI / 2, true);
+    CGPathAddArc(thepath, NULL, xstart + .5 * width, ystart + .5 * height, .5 * height, M_PI / 2, 0, true);
+    CGPathMoveToPoint(thepath, NULL, xstart + .25 * width, ystart + .5 * height);
+    CGPathAddArc(thepath, NULL, xstart + .375 * width, ystart + .5 * height, .25 * height, M_PI, 0, false);
+    CGPathMoveToPoint(thepath, NULL, xstart + .5 * width, ystart + .5 * height);
+    CGPathAddArc(thepath, NULL, xstart + .625 * width, ystart + .5 * height, .25 * height, M_PI, 0, true);
+    CGPathMoveToPoint(thepath, NULL, xstart + .4375 * width, ystart + .5 * height);
+    CGPathAddArc(thepath, NULL, xstart + .375 * width, ystart + .5 * height, .125 * height, 0, M_PI / 2, true);
+    CGPathAddArc(thepath, NULL, xstart + .375 * width, ystart + .5 * height, .125 * height, M_PI / 2, 0, true);
+    CGPathMoveToPoint(thepath, NULL, xstart + .6875 * width, ystart + .5 * height);
+    CGPathAddArc(thepath, NULL, xstart + .625 * width, ystart + .5 * height, .125 * height, 0, M_PI / 2, true);
+    CGPathAddArc(thepath, NULL, xstart + .625 * width, ystart + .5 * height, .125 * height, M_PI / 2, 0, true);
+
+    EXPECT_TRUE(testSymmetricEquivalence(thepath, CGPathCreateCopy(thepath)));
```

Comment:
It does log the line since testSymmetricEquivalence returns the result. It doesn't grab which direction the equivalence failed on but this was preferable to having two macros for fail vs succeed or having to pass an extra bool to state whether it should fail or succeed. At least in my opinion.

---
In reply to: [97699788](https://github.com/Microsoft/WinObjC/pull/1813#discussion_r97699788) [](ancestors = 97699788)